File: CS/Scaffolding.CodeFirst/Data/Department.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.ComponentModel.DataAnnotations;

namespace Scaffolding.CodeFirst.Model {
    public class Department {
        public Department() {
            Courses = new HashSet<Course>();
        }
        public int DepartmentID { get; set; }
        public string Name { get; set; }
        public virtual ICollection<Course> Courses { get; set; }
    }
}

File: VB/Scaffolding.CodeFirst/Data/Department.vb
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.ComponentModel.DataAnnotations

Namespace Scaffolding.CodeFirst.Model
	Public Class Department
		Public Sub New()
			Courses = New HashSet(Of Course)()
		End Sub
		Public Property DepartmentID() As Integer
		Public Property Name() As String
		Public Overridable Property Courses() As ICollection(Of Course)
	End Class
End Namespace